05_test.sh

```diff
@@ -139,11 +157,27 @@ check_bmh_status(){
 #
 get_vm_ip(){
   local BMH_NAME="${1}"
-  sudo virsh net-dhcp-leases baremetal | grep "${BMH_NAME}" | awk '{print $5}' \
-    | cut -d '/' -f1
+  local DHCP_RANGE
+  local DHCP_START
+  local DHCP_STOP
+  local CONFIGMAP
+
+  # Get configmap for ironic operator
+  CONFIGMAP=$(kubectl get configmap -n metal3 | grep ironic-bmo-configmap | awk '{print $1}')
+
+  # Get the DHCP start and end values
+  DHCP_RANGE=$(kubectl get configmap -n metal3 "$CONFIGMAP" -o yaml | grep "DHCP_RANGE" | awk '{print $2}')
```

Comment:
Could we pass either a go template or jsonpath to the kubectl here instead of the grep/awk filtering?

Reply:
done, thanks